### ISSUE TITLE
Arbitration ID & 0x700

### ISSUE THREAD

**Issue by remusmp** [arachnidlabs/cyflash#16]
Hi,

Thank you very much for the cyflash tool. I use it to flash over CAN bus and I managed to get it up and running. One thing I was not aware of until I looked into the protocol.py code is that the arbitration ID is filtered with 0x700. I had my PSOC with an ID that was causing cyflash to throw invalid frames. After I assigned the PSOC an ID that would successfully pass the arbitration ID filtering everything went fine.

I'm writing this because I think users should be aware of the arbitration ID filtering - maybe just adding a short line in the readme.

Thanks. R.

**Comment by IronSean** [comment]
Sorry, this is in this repo because I was new to git and didn't realize
pull requests kept to the head of the branch. That was a bit of code
specific to my uses, which I added after generating a pull request. That id
check can and should be safely removed.

I added it because in our uses we have other CANbus devices which still
communicate while a bootload operation is happening which was looking up
the bootloader.

A safer solution might be to filter based on the id which was input into
the execution call. I'll try and fix and add a PR removing this.

Sean
